release_notes.md

```diff
@@ -1,3 +1,50 @@
+**********************************
+# Release Notes for OpenMDAO 3.6.0
+
+January 14, 2020
+
+OpenMDAO 3.6.0 provides new features, several bug fixes, as well as
+documentation and visualization updates.
+
+Thank you to users @Dakror and @cfe316 for contributing to this release.
+
+## Backwards Incompatible API Changes:
+
+- None
+
+## Backwards Incompatible NON-API Changes:
+
+- None
+
+## New Features:
+
+- (POEM_032) Implemented a new scaling report feature that is intended to provide the user with some information as to how well their problem is scaled. [#1820](https://github.com/OpenMDAO/OpenMDAO/pull/1820)
+- (POEM_036) Implemented Serialization of Kriging training weights [#1830](https://github.com/OpenMDAO/OpenMDAO/pull/1830)
```

Comment:
I think for this, we should say "Implemented caching of the Kriging training weights by serializing them and saving them to a file so they can be used in later runs to save computation time."

Reply:
done